src/library/Validate.js

```diff
@@ -3,8 +3,24 @@
 /* eslint-disable no-unused-vars */
 const VALID_AGGREGATORS = ['subtract', 'add', 'first', 'last', 'max', 'min', ''];
 const VALID_BOOLEANS = ['and', 'or'];
-const VALID_OPERATORS = ['equal', 'notequal', 'lessthan', 'greaterthan'];
-const VALID_OPERANDS = ['input', 'contracts.label', 'contracts.contract_name', 'addresses.address', 'addresses.label'];
+const VALID_OPERATORS = [
+  'equal',
+  'notequal',
+  'lessthan',
+  'greaterthan',
+  'lessthanorequal',
+  'greaterthanorequal',
+];
+const VALID_OPERANDS = [
+  'input',
+  'contract_label',
+  'contract_name',
+  'contract_address',
+  'addresses.address',
```

Comment:
I don't think we need this line (`addresses.address`) since it's equivalent to `contract_address`. Maybe a leftover from the fix?

Reply:
And apologies for not catching this sooner!

Reply:
Good call, I might copied the wrong SQL string in query_field_type file.

Reply:
Can I merge this?